importer/src/powdr_std.rs

```diff
@@ -97,13 +97,17 @@ impl Folder for StdAdder {
                     }
                 }
                 .map(|value| ModuleStatement::SymbolDefinition(SymbolDefinition { value, ..d })),
+                ModuleStatement::TraitImplementation(trait_impl) => {
+                    self.fold_trait_implementation(trait_impl).map(From::from)
+                }
             })
             .collect::<Result<Vec<_>, _>>()?;
 
         // Check whether the module already has a definition for `std`
         // (E.g. the main module)
         let has_std = statements.iter().any(|s| match s {
             ModuleStatement::SymbolDefinition(d) => d.name == "std",
```

Comment:
Would it work to use `defined_names` here?

Reply:
Isn't there a function on ModuleStatement that returns the names defined by the statement? if yes, I think it would be cleaner to use that function instead of relying on an "internal" property of ModuleStatement::SymbolDefinition.